hat/menupermissions/models.py

```diff
@@ -17,46 +17,75 @@
 `/api/permissions/` endpoint
 """
 
-
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
-MODIFICATIONS = _("Modifications")
+_ASSIGNMENTS = "iaso_assignments"
```

Comment:
Didn't you remove translations here. _() is used to translate the presentation text of a permission, normally

Reply:
I have moved the translation to the list and used the constants for the keys.